services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/[asset]/index.js

```diff
@@ -331,9 +178,40 @@ const Asset = ({ libraryData, params }) => {
 
 Asset.propTypes = {
   libraryData: libraryPropTypes,
+  overviewMdxSource: PropTypes.shape({
+    compiledSource: PropTypes.shape({
+      value: PropTypes.string,
+      data: PropTypes.shape({
+        matter: PropTypes.object
+      })
+    }),
+    mdxError: PropTypes.shape({
+      name: PropTypes.string,
+      message: PropTypes.string,
+      stack: PropTypes.string,
+      position: PropTypes.string
+    }),
+    warnings: PropTypes.arrayOf(PropTypes.string)
+  }),
   params: paramsPropTypes
 }
 
+const getOverviewMdxSource = async (assetData, libraryData) => {
```

Comment:
This function could benefit from some JSDoc or inline comments.

Reply:
Done!